packages/ui-library/src/elements/table/style.css

```diff
@@ -1,15 +1,55 @@
 @layer components {
 	.yst-root {
 		.yst-table-wrapper {
-			@apply yst-rounded-lg;
+			table {
+				@apply yst-min-w-full;
+
+				.yst-table-header {
+					@apply yst-px-3 yst-py-4 yst-text-start yst-text-sm yst-font-semibold yst-text-slate-900;
+				}
+
+				.yst-table-cell {
+					@apply yst-px-3 yst-py-4 yst-text-sm yst-text-slate-600;
```

Comment:
I think this `py-4` should be a little less for the minimal. In the design it seems to be 17px between the text. Which is around half? 🤔 
I can't get it to show the distance to the separator.

Maybe something like this in the minimal?
```css
				.yst-table-header, .yst-table-cell {
					@apply yst-py-2;
				}
```